scripts/tosdr-topics.json

```diff
@@ -7,7 +7,8 @@
         "tracks you on other websites",
         "logs are kept forever",
         "they can use your content for all their existing and future services",
-        "your data is used for many purposes"
+        "your data is used for many purposes",
+        "device fingerprinting"
```

Comment:
This was added by TOSDR in the past couple of days. I thought it's worth adding it here.